changelog/2024-04-30-conduit-0-10-0-release.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: '2024-04-30-conduit-0-10-0-release'
+title: Conduit 0.10.0 release
+draft: false
+tags: [conduit, release]
+---
+
+[**Conduit v0.10**](https://github.com/ConduitIO/conduit/releases/tag/v0.10.0) is here! This release introduces **native support for multiple collections**, addressing user feedback and enhancing the flexibility of data integration.
+
+<!--truncate-->
+
+### Key Highlights
```

Comment:
The HTTP connector also has a programmable source. Technically, it was released later, but it might be worth mentioning.

Reply:
I think we should honor the timeline as if these were added in order. I have another changelog for the HTTP connector, so hopefully that would do the job! Let me know if you disagree.

Reply:
Nope, you're completely right about that. :+1: